### PR TITLE
Title Block

### PR DESCRIPTION
**Pull request by andicarlstrom**
Title block with basic variant for price tag. Defs think there could be some improvements on margins and padding on the pricetag and the title. 

Test URLs:
- Before: [https://main--site--normal-icecream.aem.live/drafts/title](https://main--site--normal-icecream.aem.live/drafts/title)
- After: [https://feat-title--site--normal-icecream.aem.live/drafts/title](https://feat-title--site--normal-icecream.aem.live/drafts/title)

### CODE REVIEW

File: blocks/title/title.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    overflow: hidden;
+}
+
+.title .title-price::before {
+    content: '';
+    position: absolute;
+    background-image: url('../../icons/pricetag.svg');
```
Comment (fkakatie) [inline blocks/title/title.css:49]: would prefer to add the svg in the dom directly than rely on a relative path in the css.

File: icons/pricetag.svg
Comment (fkakatie) [inline icons/pricetag.svg:1]: the way this file is written will always set the border color and fill. is this what we want? or should the color be inferred from content and context (able to be influenced by setting stroke or fill in css)?

File: blocks/title/title.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    border-bottom: 2px solid var(--blue);
+    position: relative;
+    margin-top: 80px;
+    margin-bottom: 40px;
+}
+
+.title h2 {
+    line-height: 1;
+    margin: 1px 0 0;
+    overflow-wrap: break-word;
```
Comment (fkakatie) [inline blocks/title/title.css:17]: i think this might be worth setting in styles for all headings

File: blocks/title/title.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+export default function decorate(block) {
+  const variants = [...block.classList];
+  [...block.children].forEach((row) => {
+    // TODO - need to add counter to title, should be set up when doing Square integration
```
Comment (fkakatie) [inline blocks/title/title.js:4]: agreed!

File: blocks/title/title.css [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-xs);
+    line-height: 24px;
+}
+
+.title .title-price-long-subtext {
+    text-align: center;
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-xxs);
+    line-height: 24px;
```
Comment (fkakatie) [inline blocks/title/title.css:95]: most of this is duplicative. is there a way to style all the subtext in one declaration, then update the font size in one line in another?

File: blocks/title/title.css (hunk 2 above)
Comment (andicarlstrom) [reply blocks/title/title.css]: Added this to styles for headers :) Great feedback

File: blocks/title/title.css (hunk 1 above)
Comment (andicarlstrom) [reply blocks/title/title.css]: Fixed, I replaced the old logic with adding the svg element to the dom directly.

File: icons/pricetag.svg
Comment (andicarlstrom) [reply icons/pricetag.svg:1]: Fixed :)

**Pull request merged**